spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -557,31 +556,31 @@ at the runtime.
 ----
 ---
 applications:
-- name: test-server
-  host: test-server
-  memory: 1G
-  disk_quota: 1G
+- name: data-flow-server
+  host: data-flow-server
+  memory: 2G
+  disk_quota: 2G
   instances: 1
-  path: spring-cloud-dataflow-server-cloudfoundry-VERSION.jar
+  path: {ABSOLUTE PATH TO SERVER UBER-JAR}
```

Comment:
I'll find out, that is what was listed previously in the getting started section for the manifest.  I think relative also works.